components/ToolBar.jsx

```diff
@@ -1,56 +1,96 @@
-import React from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import { setSelectedVariables, setPlotType } from '../actions/selectedVariablesActions';
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { setSelectedVariables, setPlotType } from "../actions";
+import { useIntl } from "react-intl";
 
 const Toolbar = () => {
-  const data = useSelector(state => state.root.file.fileData);
-  const selectedVariables = useSelector(state => state.root.selectedVariable);
-  const plotType = useSelector(state => state.root.plotType);
+  const intl = useIntl();
+  const data = useSelector((state) => state.root.file.fileData);
+  const selectedVariables = useSelector((state) => state.root.selectedVariable);
+  const plotType = useSelector((state) => state.root.plotType);
   const dispatch = useDispatch();
 
   const columns = data.length > 0 ? Object.keys(data[0]) : [];
-  
+
   const handleVariableChange = (e, index) => {
     const newSelectedVariables = [...selectedVariables];
     newSelectedVariables[index] = e.target.value;
     dispatch(setSelectedVariables(newSelectedVariables));
   };
 
-  const handlePlotTypeChange = e => {
+  const handlePlotTypeChange = (e) => {
     dispatch(setPlotType(e.target.value));
   };
 
   return (
-    <div>
-      <h4>Toolbar</h4>
-      <div>
-        <label>Plot Type: </label>
-        <select value={plotType} onChange={handlePlotTypeChange}>
-            <option value="">Select Plot Type</option>
-            <option value="bar">Bar Plot</option>
-            <option value="scatter">Scatter Plot</option>
+    <div style={styles.container}>
+      <h4 style={styles.title}>{intl.formatMessage({ id: "toolbar.title" })}</h4>
+      <div style={styles.row}>
+        <label>{intl.formatMessage({ id: "toolbar.plotType" })}: </label>
+        <select style={styles.select} value={plotType} onChange={handlePlotTypeChange}>
+          <option value="">
+            {intl.formatMessage({ id: "toolbar.selectPlotType" })}
+          </option>
+          <option value="bar">
+            {intl.formatMessage({ id: "toolbar.barPlot" })}
+          </option>
+          <option value="scatter">
+            {intl.formatMessage({ id: "toolbar.scatterPlot" })}
+          </option>
         </select>
-        </div>
-        {plotType && (
+      </div>
+      {plotType && (
         <>
-            <div>
-            <label>Variable 1: </label>
-            <select onChange={(e) => handleVariableChange(e, 0)}>
-                <option value="">Select Variable</option>
-                {columns.map((col, index) => <option key={index} value={col}>{col}</option>)}
+          <div style={styles.row}>
+            <label>{intl.formatMessage({ id: "toolbar.variable1" })}: </label>
+            <select style={styles.select} onChange={(e) => handleVariableChange(e, 0)}>
```

Comment:
For `handleVariableChange(e, 0/1)`, you can define an enum for the constant 0/1 values so the code is more readable. Also, we prefer the function call in line 30 above instead of using an inline function call. Thanks!